Zowe process documents for review by the Zowe ZLC on 20180829

jmertic opened the pull request:
Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Zowe process documents for review by the Zowe ZLC on 20180829

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Filled out the release.md and cleaned up sections of the other documents

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jplinardon reviewed (commented):
The date seems incorrect: for a term that will end by "July 31, 2018"

jplinardon reviewed (commented):
in process/proposal_process.md:

possible grammatical issue: "to governed"

The release process section should be customized to indicate that we have a CI/CD squad that is responsible for coordinating releases, with approval and support from other squads.
Release Process
Project releases will occur on a scheduled basis as agreed to by the maintainers.

jplinardon reviewed (commented):
in process/roles.md
typo: defintion

jmertic committed:
Fixed the date

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Fixed typo

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Fixed typo

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Updated Release Process section.

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

Tbr00ksy committed:
Update README.md

Tbr00ksy committed:
Update example_governance.md

Tbr00ksy committed:
Update roles.md

Tbr00ksy committed:
Added Content

Added content to the Zowe Release Process.
-s

Tbr00ksy committed:
Updated Proposal Process

Merged the "stages" file: https://github.com/zowe/zlc/blob/forreviewzlc20180829/process/stages.md
Made additional edits:
- Submit a ZLC issue for a new sub-project proposal
- Revised sub-project proposal requirements
- Revised Active Stage acceptance criteria
-

Tbr00ksy committed:
Fixed Typo

Tbr00ksy committed:
Merged into Process

This file can be deleted as the content has been merged into the Sub-Project Proposal Process here: 
https://github.com/zowe/zlc/blob/forreviewzlc20180829/process/proposal_process.md

Tbr00ksy committed:
Updates Zowe Leader Role

Changed worded to reflect what was agreed to in the Zowe project proposal.

jmertic committed:
Updated for current guidance.

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
fixed typo

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Fixed link

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

nannanli reviewed (commented):
Hi John,

I'm from the CI/CD squad. I just reviewed this document and have some questions/comments:
1. In README.md ->current ZLC voting members, now it's just a list of names. Shall we add some short bio for each one and add a picture for them?
2. Example_governance.md has some content that overlaps with what's in role.md (which defines the project roles like adopter, committer, and contributor, leads) and release.md (which defines release process, communication chanels, etc). Shall we consider combining these two documents into one?

Also Roles.md -> Contributor describes how to contribute with details like DCO. This part of information is not definition but part of the how to contribute process. So I'd suggest we combine them and add links to contribution guidelines. 

Thanks!

Tbr00ksy committed:
Minor Edits from Ashley

Thank you @nannanli for the edits.

Tbr00ksy committed:
Minor Edits

Thank you @nannanli for the edits.

Tbr00ksy committed:
Minor Edits

Thank you @nannanli for the edits.

Tbr00ksy committed:
Fixed Links and Minor Edits

Thank you @nannanli for the edits.

Tbr00ksy committed:
Minor Edits

Thank you @nannanli for the edits.

hogstrom committed:
Minutes and links to recording from the 2018-09-19 ZLC meeting.

Signed-off-by: Matt Hogstrom <matt@hogstrom.org>

1000TurquoisePogs commented on README.md:
```diff
@@ -1,2 +1,40 @@
+
+- [Contribution Guidelines](https://zowe.org/contribute/)
+- [Project Role Definitions](process/roles.md)
+- [Code of Conduct](https://www.linuxfoundation.org/code-of-conduct/) - Zowe community members are bound to the Linux Foundation Code of Conduct.
```
The very same CoC that is causing drama and threats within the Linux community? Let's avoid this until the Linux community at large settles in one direction or another. I don't want Zowe to get caught in the middle of controversy.

jmertic replied:
Entirely different one - we use this for LF events and has existed long before the kernel community has looked into establishing one.

hogstrom replied:
This looks like the CoC referenced is for conferences.  I don't see an issue with it.  Honestly, the code of conduct in its current form focus a LOT on sexual issues and I think tries to hard but the intent seems fine to me.  I think we let it go.

1000TurquoisePogs replied:
Code of conducts are important to get right because they establish a culture that surrounds a project. As you said, that one seems more suited to conferences than code. Here's an alternative CoC which I found to be a good fit, from KDE: https://www.kde.org/code-of-conduct/

jmertic replied:
If this is a large concern, I'd suggest raising to the Open Mainframe Project TSC.

Tbr00ksy replied:
@1000TurquoisePogs Are you OK with the current COC reference? I'd like to merge this PR to master and close out the waffle ticket.

jmertic committed:
Merge branch 'forreviewzlc20180829' of https://github.com/zowe/zlc into forreviewzlc20180829

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Add CII badge status

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Cleaned up

Signed-off-by: John Mertic <jmertic@linuxfoundation.org>

jmertic committed:
Merge branch 'forreviewzlc20180829' of https://github.com/zowe/zlc into forreviewzlc20180829

Signed-off-by:  <>
